dango/cli/src/start.rs

```diff
@@ -74,9 +75,12 @@ impl StartCmd {
     }
 
     /// Run the HTTP server
-    async fn run_httpd_server(database_url: String) -> anyhow::Result<()> {
-        indexer_httpd::server::run_server(None, None, database_url, config_app, build_schema)
-            .await?;
+    async fn run_httpd_server(context: Context) -> anyhow::Result<()> {
+        if let Err(err) =
+            indexer_httpd::server::run_server(None, None, context, config_app, build_schema).await
+        {
+            tracing::error!("Failed to run HTTP server: {:?}", err);
+        }
 
         Ok(())
     }
```

Comment:
@penso It looks like now we only prints the error but don't abort. This function always returns ok. Is this intended?

Reply:
It is intended, my biggest fear would be the node not to start because of some side issues like the local indexer http so I'll try to make it loose and not prevent starting.